config/app_test_default.js

```diff
@@ -1,7 +1,8 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Is this supposed to stay here?

Reply:
It was just part of the transpilation from the .ts version to the .js version - I will remove.